lib/examples/ext_ctrls_test/main.rs

```diff
@@ -0,0 +1,29 @@
+use nix::fcntl::{open, OFlag};
```

Comment:
I wonder if this example shouldn't use the safe API directly? Examples are mainly tailored to how end users should use a crate, and that will be through the safe API.

Reply:
The example does use the safe abstractions provided by this PR. However, opening device special files happens in an `unsafe` block, which is not the preferred way. I will change it in the next version.